### ISSUE TITLE
Add Credits to bottom

### ISSUE THREAD

**Issue by ghost** [gytdau/tome#12]
Would be indeed very nice if there would be a credits tab at the bottom. :sagittarius:

**Comment by gytdau** [comment]
Who should be in the tab? If someone commits 1 change, something as small as a typo fix, should they still go in there?

**Comment by ghost** [comment]
No. I'm talking original founder/creators. Additionally, you could have a small part of the site allocated to the people who helped out. :sagittarius:

**Comment by gytdau** [comment]
I don't think there's enough [contributors.](https://github.com/gytdau/tome/graphs/contributors?from=2015-12-27&to=2016-02-01&type=c) It would be a good idea if we had some more contributors though.

**Comment by ghost** [comment]
Then add the original founders until you get a few contributors on and going. :sagittarius:
